bigbluebutton-html5/imports/ui/components/dropdown/component.jsx

```diff
@@ -83,15 +83,18 @@ class Dropdown extends Component {
 
   handleShow() {
     this.setState({ isOpen: true }, this.handleStateCallback);
-
-    const contentElement = findDOMNode(this.refs.content);
-    contentElement.querySelector(FOCUSABLE_CHILDREN).focus();
   }
 
   handleHide() {
+
+    const { autoFocus } = this.props;
+
     this.setState({ isOpen: false }, this.handleStateCallback);
-    const triggerElement = findDOMNode(this.refs.trigger);
-    triggerElement.focus();
+
+    if (autoFocus) {
```

Comment:
Did you tried using the `autofocus` property of the HTML5 spec? I'm not sure but for buttons and inputs it works on most browsers.